docs/ga-roadmap.md

```diff
@@ -1 +1,54 @@
 # Collector v1 Roadmap
+
+This document contains the roadmap for the Collector as of 4-Apr-2024. The main goal of this roadmap is to provide clarity on the areas of focus in order to release a v1 of the Collector.
```

Comment:
nit: One can figure out the date by looking at the last commit that updated this file
```suggestion
This document contains the roadmap for the Collector. The main goal of this roadmap is to provide clarity on the areas of focus in order to release a v1 of the Collector.
```